snippets/getImages.md

```diff
@@ -0,0 +1,17 @@
+### getImages
+
+Fetches all images from within an element and puts them into an array
+
+Use `Element.prototype.getElementsByTagName()` to fetch all `<img>` elements inside `Element`, `Array.prototype.map()` to map every `src` attribute of their respective `<img>` element. `New Set()` creates a `Set` of unique images which is then converted into an `Array` with the `...` operator.
```

Comment:
And you don't have to mention how you declare the Set as we all know how to write `New Set()` IMHO 🙂 so just "Then, create a `Set` to eliminate duplicates and return the array from it".